src/components/EmojiPicker/EmojiPickerMenu/BaseEmojiPickerMenu.tsx

```diff
@@ -108,7 +82,10 @@ function ListEmptyComponent() {
     return <Text style={[styles.textLabel, styles.colorMuted]}>{translate('common.noResultsFound')}</Text>;
 }
 
-function BaseEmojiPickerMenu({headerEmojis, scrollToHeader, isFiltered, listWrapperStyle, forwardedRef, data, renderItem, stickyHeaderIndices, extraData, alwaysBounceVertical}) {
+function BaseEmojiPickerMenu(
+    {headerEmojis, scrollToHeader, isFiltered, listWrapperStyle = [], data, renderItem, stickyHeaderIndices = [], extraData = [], alwaysBounceVertical = false}: BaseEmojiPickerMenuProps,
+    forwardedRef: LegacyRef<FlashList<Emoji | HeaderEmoji | EmojiSpacer>>,
```

Comment:
same here I will rename it to just `ref` and why we are using `LegacyRef` type here? I think it should be `ForwardedRef`